config/header.html

```diff
@@ -7,3 +8,24 @@
   });
   }}();
 </script>
+<style>
+.deluge {
+  background-color: #13aa52;
+}
+.deluge .switch.good.selected, .deluge .switch.good.selected:hover {
+  color: #13aa52;
+}
```

Comment:
These are inlined, as reporting from relative paths is nigh impossible and I can only add one additional css file via javadoc.